### PR TITLE
Enabling site selector in PageTree

### PR DESCRIPTION
**Pull request by hbaptiste**
Enabling site selector in PageTree

### CODE REVIEW

File: src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig [hunk 1]
```diff
@@ -1,3 +1,8 @@
+{% set title = data.title %}
+{% if data.siteLabel %}
+    {% set title = data.title + " (" + data.siteLabel + ")" %}
```
Comment (mickaelandrieu) [inline src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig]: this syntax is not valid in twig. The string concatenation is done with `~` operator.

File: src/tb/component/siteselector/main.js [hunk 2]
```diff
@@ -61,24 +64,35 @@ define(
...
+                        if (verbose) {
+                            selectedSite = underscore.findWhere(sites, {uid: selectedSite});
+                        }
+                        return selectedSite;
                     },
 
                     getJqueryElement: function () {
                         return jQuery(this.widget);
                     }
+
```
Comment (mickaelandrieu) [inline src/tb/component/siteselector/main.js]: can you remove theses lines  ?

File: src/tb/component/siteselector/main.js [hunk 3]
```diff
@@ -61,24 +64,35 @@ define(
...
                         });
 
                         return this.widget;
                     },
 
-                    getSelectedSite: function () {
-                        return jQuery(this.widget).find("select").eq(0).val();
+                    getSelectedSite: function (verbose) {
+                        var selectedSite = jQuery(this.widget).find("select").eq(0).val();
+                        if (verbose) {
```
Comment (mickaelandrieu) [inline src/tb/component/siteselector/main.js]: use strict comparaison here :)

File: src/tb/component/treeview/PopinTreeView.js [hunk 4]
```diff
@@ -59,6 +60,13 @@ define(
                     }
                 },
 
+                handleCreate: function () {
+                    if (typeof this.options.onCreate === 'function') {
+                        this.options.onCreate.call(this);
+                    }
+                },
+
+
```
Comment (mickaelandrieu) [inline src/tb/component/treeview/PopinTreeView.js]: one line to be removed here.

File: src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig (hunk 1 above)
Comment (hbaptiste) [inline src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig]: It's not twig but nunjucks. And it's valid.

File: src/tb/component/siteselector/main.js (hunk 3 above)
Comment (hbaptiste) [inline src/tb/component/siteselector/main.js]: Why?

File: src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig (hunk 1 above)
Comment (mickaelandrieu) [inline src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig]: don't use the twig extension then :p

File: src/tb/component/siteselector/main.js (hunk 3 above)
Comment (mickaelandrieu) [inline src/tb/component/siteselector/main.js]: is verbose a string, a boolean , an int ? we don't know at this moment.

File: src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig (hunk 1 above)
Comment (hbaptiste) [inline src/tb/component/formbuilder/form/element/templates/nodeSelector_item.twig]: you'll have to change to whole project to fix that.

**Comment by fkroockmann** [comment]
:+1:

**Comment by fkroockmann** [comment]
Can you squash plz ?

**Pull request merged**